db/carrier_icons.yml

```diff
@@ -2,11 +2,52 @@
   name: ups
   label: UPS
   group: other
+-
```

Comment:
Can you supply a usps_symbol.svg variant? Just the symbol without "USPS" text

![CleanShot 2024-02-02 at 12 29 13@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/4adff7a5-3ede-4c53-a9c0-a65468b82216)